consensus/src/simplex/actors/voter/state.rs

```diff
@@ -204,23 +219,15 @@ impl<E: Clock + Rng + CryptoRng + Metrics, S: Scheme<D>, D: Digest> State<E, S,
         notarization: Notarization<S, D>,
     ) -> (bool, Option<S::PublicKey>) {
         let view = notarization.view();
-        let seed = self
-            .scheme
-            .seed(notarization.round(), &notarization.certificate);
-        let added = self.create_round(view).add_notarization(notarization);
-        self.enter_view(view.next(), seed);
-        added
+        self.enter_view(view.next(), Some(&notarization.certificate));
+        self.create_round(view).add_notarization(notarization)
```

Comment:
The order of these operations is switched to avoid having to clone the certificate, AFAIK there are no issues with switching these since `enter_view` will operate on the next view's round, but can revert and just clone it.